plugins/elasticsearch/elasticsearch.go

```diff
@@ -166,43 +212,66 @@ func (o *ElasticSearchOutput) receiver(or OutputRunner, wg *sync.WaitGroup) {
 				outBatch = append(outBatch, outBytes...)
 				if count = count + 1; o.bulkIndexer.CheckFlush(count, len(outBatch)) {
```

Comment:
Since `count` is now being manipulated from multiple goroutines, we have to start using the synchronized `atomic` API for all access and mutation.